lib/VCMIDirs.cpp

```diff
@@ -348,6 +349,7 @@ std::vector<bfs::path> VCMIDirsWIN32::dataPaths() const
 }
 
 bfs::path VCMIDirsWIN32::clientPath() const { return binaryPath() / "VCMI_client.exe"; }
+bfs::path VCMIDirsWIN32::editorPath() const { return binaryPath() / "VCMI_editor.exe"; }
```

Comment:
There will be template editor and campaign editor at some point
```suggestion
bfs::path VCMIDirsWIN32::mapEditorPath() const { return binaryPath() / "VCMI_editor.exe"; }
```

Reply:
Will change, although I think that it would be better to have such tools integrated into our map editor.

Reply:
Maybe a good option instead of making many apps. So agree now with your option

Reply:
Well, for now this thing is still only *map* editor, so renames (both here and UI button) make sense. Can be reverted later after editor upgrade